### DISCUSSION TITLE
p/0e1fad3617514ced89c07959c8d8f52b/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#59] [Announcements]
# p/0e1fad3617514ced89c07959c8d8f52b/

动机自从更新 nodejs@18 并切换到 esm only 以来，许多库都已经被替换成支持 esm 导入，但其中 fs-extra 却一直没有正确的支持 esm 使用，也没有找到合适的替代品。在吾辈之前提出的 一个 PR 被否定之后，决定重新发布一个正确支持 esm 使用的 fs-extra-unified 模块。 如果你还不知道 fs-extra 是什么，这里可以简单介绍一下：它是一个 nod

https://blog.rxliuli.com/p/0e1fad3617514ced89c07959c8d8f52b/

**Comment by real-rain** [comment]
大佬，我有一个内网项目集成的fs-extra是7.0.1比较老，未支持esm，可否通过该脚本转换

**Reply by rxliuli** [reply]
更新 fs-extra 为最新版？

**Reply by real-rain** [reply]
在内网环境，无法更新上传
